src/Options.js

```diff
@@ -65,27 +66,26 @@ export var optionsV01 = enumerableOnlyObject({
 export var versionLockedOptions = optionsV01;
 
 // Options are just a plain old object. There are two read only views on this
-// object, parseOptions and transformOptions.
+// object, parseView() and transformView().
 //
 // To set an option you do `options.classes = true`.
 //
 // An option value is either true, false or a string. If the value is set to
-// the string "parse" then the transformOption for that option will return
-// false. For example:
+// the string "parse" then the transformView(optionName) for that option
+// will return false. For example:
 //
 //   options.destructuring = 'parse';
-//   parseOptions.destructuring === true;
-//   transformOptions.destructuring === false;
+//   parseView('destructuring') === true;
```

Comment:
I don't like that this is a function. Making it a "proxy" object leads to a cleaner API.